### DISCUSSION TITLE
BEAMnrc simulation issue: Getting errors in BEAMnrc

### DISCUSSION THREAD

**Discussion by physicsmed** [nrc-cnrc/EGSnrc#998] [Questions]
Hello researchers!
i am trying to get a phase space file from BEAMnrc simulation, but the simulation takes just 5 min to be finished and it creates file error in BEAM_mylinac folder conatain these following lines and i hope to solve this issue :

"If you are not trying to reset transport parameters,
  ignore all the output until the message
  ******************** end input transport parameter ***********************

 ***************ERROR***************
 >>Radiative Compton corrections<< NOT FOUND
 END OF DELIMETER: MC TRANSPORT PARAMETER
 ***************ERROR***************
 >>Triplet production<< NOT FOUND
 END OF DELIMETER: MC TRANSPORT PARAMETER
 ***************ERROR***************
 >>scale L EII cross-sections<< NOT FOUND
 END OF DELIMETER: MC TRANSPORT PARAMETER
 ***************ERROR***************
 >>Electric Field<< NOT FOUND
 END OF DELIMETER: MC TRANSPORT PARAMETER
 ***************ERROR***************
 >>Magnetic Field<< NOT FOUND
 END OF DELIMETER: MC TRANSPORT PARAMETER
 ***************ERROR***************
 >>EM ESTEPE<< NOT FOUND
 END OF DELIMETER: MC TRANSPORT PARAMETER
 ***************ERROR***************
 >>Photonuclear attenuation<< NOT FOUND
 END OF DELIMETER: MC TRANSPORT PARAMETER
 ***************ERROR***************
 >>Photonuclear cross sections<< NOT FOUND
 END OF DELIMETER: MC TRANSPORT PARAMETER
 ***************ERROR***************
 >>Set ECUT<< NOT FOUND
 END OF DELIMETER: MC TRANSPORT PARAMETER
 ***************ERROR***************
 >>Set ECUT start region<< NOT FOUND
 END OF DELIMETER: MC TRANSPORT PARAMETER
 ***************ERROR***************
 >>Set ECUT stop region<< NOT FOUND
 END OF DELIMETER: MC TRANSPORT PARAMETER
 ***************ERROR***************
 >>Set PCUT<< NOT FOUND
 END OF DELIMETER: MC TRANSPORT PARAMETER
 ***************ERROR***************
 >>Set PCUT start region<< NOT FOUND
 END OF DELIMETER: MC TRANSPORT PARAMETER
 ***************ERROR***************
 >>Set PCUT stop region<< NOT FOUND
 END OF DELIMETER: MC TRANSPORT PARAMETER
 ***************ERROR***************
 >>Set SMAX<< NOT FOUND
 END OF DELIMETER: MC TRANSPORT PARAMETER
 ***************ERROR***************
 >>Set SMAX start region<< NOT FOUND
 END OF DELIMETER: MC TRANSPORT PARAMETER
 ***************ERROR***************
 >>Set SMAX stop region<< NOT FOUND
 END OF DELIMETER: MC TRANSPORT PARAMETER

  ******************** end input transport parameter ***********************

 ******************ERROR***********************
 END OF FILE REACHED BUT VALUE SOUGHT NOT FOUND
 PROBABLY A MISSING/MISSPELLED START DELIMETER
 VALUE SOUGHT: >>PHSP OUTPUT DIRECTORY<<
 START DELIMETER: >>:START USER INPUTS:<<"

**Comment by rtownson** [comment]
Try loading the input file in beamnrc_gui, making sure all the parameters are set, and saving the input file. The GUI may fix any mistakes for you. If you are still having problems, post your input file here, and/or the .egslog or terminal output.

**Reply by physicsmed** [reply]
firstly, i would thank you for your interest and for the answer.
i have run it in beamnrc_gui and it creates the same error file with same errors as i wrote it previously, but graphically it shows this message:

"  
...
...
Rayleigh data available for medium  7 in PEGS4 data set. 
 
  ->  100 atomic ff values computed! 
      preparing data for Rayleigh sampling ... done 
 
 Rayleigh data available for medium  8 in PEGS4 data set. 
 
  ->  100 atomic ff values computed! 
      preparing data for Rayleigh sampling ... done 
 
 Found medium with gas pressure 
 
***************** Error:  
 IN HATCH: REQUESTED RAYLEIGH OPTION FOR MEDIUM  1 
 BUT RAYLEIGH DATA NOT INCLUDED IN PEGS4 FILE. 
 YOU WILL NOT BE ABLE TO USE THE PEGS4 DATA WITH RAYLEIGH ON! 
 
***************** Quiting now.  ""

i think that the problem is in PEG4, do you have any suggestions to solve this problem?
NOTE: the egslog it wasn't created.

**Reply by rtownson** [reply]
Right, the problem is that the PEGS4 data wasn't generated to include rayleigh data. You could generate the material again using egs_gui and turn this option on, if the physics of rayleigh scattering is important for your problem.
